src/orquestra/sdk/_base/_in_process_runtime.py

```diff
@@ -20,6 +33,30 @@
 ArtifactValue = t.Any
 TaskOutputs = t.Tuple[ArtifactValue, ...]
 
+IDS: t.Optional[t.Tuple[WorkflowRunId, ir.TaskInvocationId, TaskRunId]] = None
```

Comment:
**non-blocking,nitpick**: I think using globals here is indeed the correct, pragmatic approach. 

By convention, `ALL_CAPS` variables are constants. Meanwhile, the run identifiers change all the time. Would this be a better name?
```suggestion
current_run_ids: t.Optional[t.Tuple[WorkflowRunId, ir.TaskInvocationId, TaskRunId]] = None
```

Reply:
Good idea, [34f7c7b](https://github.com/zapatacomputing/orquestra-workflow-sdk/pull/150/commits/34f7c7b4b799dc2658cc0691577a61c38d606dd3)